apps/web/app/api/v1/management/responses/lib/response.test.ts

```
@@ -0,0 +1,347 @@
+import { validateInputs } from "@/lib/utils/validate";
+import { Organization, Prisma, Response as ResponsePrisma } from "@prisma/client";
+import { beforeEach, describe, expect, test, vi } from "vitest";
+import { prisma } from "@formbricks/database";
+import { logger } from "@formbricks/logger";
+import { DatabaseError, ResourceNotFoundError } from "@formbricks/types/errors";
+import { TResponse, TResponseInput } from "@formbricks/types/responses";
+import { getContactByUserId } from "./contact";
+import { createResponse, getResponsesByEnvironmentIds } from "./response";
+
+// Mock Data
+export const environmentId = "test-environment-id";
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)